x-pack/plugins/synthetics/public/legacy_uptime/pages/monitor_management/invalid_api_key_callout.tsx

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiButton, EuiCallOut, EuiLink, EuiSpacer } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { useEnablement } from '../../components/monitor_management/hooks/use_enablement';
+
+export const InvalidApiKeyCalloutCallout = () => {
+  const { enablement, enableSynthetics, invalidApiKeyError } = useEnablement();
+
+  if (!invalidApiKeyError || !enablement.isEnabled) {
+    return null;
+  }
+
+  return (
+    <>
+      <EuiCallOut title={API_KEY_MISSING} color="warning" iconType="help">
+        <p>{CALLOUT_MANAGEMENT_DESCRIPTION}</p>
+        {enablement.canEnable ? (
+          <EuiButton
+            fill
+            color="primary"
+            onClick={() => {
+              enableSynthetics();
+            }}
+          >
+            {SYNTHETICS_ENABLE_LABEL}
+          </EuiButton>
+        ) : (
+          <p>
+            {CALLOUT_MANAGEMENT_CONTACT_ADMIN}{' '}
+            <EuiLink href="#" target="_blank">
+              {LEARN_MORE_LABEL}
+            </EuiLink>
+          </p>
+        )}
+      </EuiCallOut>
+      <EuiSpacer size="s" />
+    </>
+  );
+};
+
+const LEARN_MORE_LABEL = i18n.translate(
+  'xpack.synthetics.monitorManagement.manageMonitorLoadingLabel.callout.invalidKey',
+  {
+    defaultMessage: 'Learn more.',
```

Comment:
```suggestion
    defaultMessage: 'Learn more',
```
We usually don't add ending punctuation after 'Learn more', a bit like a button.